pkg/options/options.go

```diff
@@ -126,6 +126,16 @@ func ObjectBucketProvisionerName() string {
 	return SubDomainNS() + "/obc"
 }
 
+// WatchNamespace returns the namespace which NooBaa operator will watch for changes
+func WatchNamespace() string {
+	ns, err := util.GetWatchNamespace()
+	if err == nil {
+		return ns
+	}
+
+	return Namespace
```

Comment:
I don't understand 😅 . Can you please elaborate?

This function is used in the operator manager to get which namespace to watch. Is it pointing out that it will be problematic if `WATCH_NAMESPACE` is different from the `--namespace` value?

Reply:
ok yes, it's not clear,
the issue I'm thinking about is when an error returned from util.getWatchNamespace(), Namespace variable as default is noobaa but noobaa namespace might not even exist, right?
or am I missing something here?

Reply:
But won't the deployment fail if Namespace is invalid? WatchNamespace is used by the manager which is run inside the operator, if the operator is running the Namespace then the Namespace must exist right? Did I got that right?

Reply:
As far as I can think, the only case where they will diverge is when `WATCH_NAMESPACE="" nb <cmd> -n random`. In this case, watch namespace is `""` however `Namespace = random`. But isn't this desirable? 

Actually because of the occasional diverging nature of "Current Namespace" and "Watch Namespace" I separated them out into a variable and method respectively.